backend/answers/models.py

```diff
@@ -70,3 +70,10 @@ def update_answer(self, question_id: int, correct_answer: str):
         session.add(self)
         session.commit()
         return self
+
+    def check_answer(self, answer: str):
+        '''check what answer is correct or not'''
+        if self.correct_answer == answer:
```

Comment:
You should create more advanced system to check question. It's one of main tasks in our project